careers/devops.markdown

```diff
@@ -0,0 +1,33 @@
+---
+title: DevOps Engineer
+summary: The Haskell Foundation seeks a passionate DevOps professional to support GHC, related Haskell projects and tooling, and the HF itself.
+postedOn: 2021-12-28
+status: Open
+---
+
+# Job Details
+
+## Responsibilities
+
+* Maintain GitLab and CI infrastructure for GHC
+* Improve CI
+* Expand the CI infrastructure to include base the base Haskell ecosystem
```

Comment:
```suggestion
* Design and implement an expansion of the CI infrastructure to include key components of the Haskell ecosystem
```

I think pointing out that the hire is responsible for design is important here.

Unless you're going for `base`, specifically? But I think that's already covered in today's CI setup, actually.